packages/get-context/src/index.ts

```diff
@@ -55,8 +55,12 @@ export interface ProjectOptions {
   rootDir: string,
 }
 
+interface HookedOptions {
+  unhookedManifest?: ProjectManifest,
```

Comment:
I am not sure about this name... maybe we can come up with a better alternative

cc @aparajita @ExE-Boss

Reply:
`originalManifest`, that's what it is. The original, untouched manifest.